frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -181,16 +181,23 @@ action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected br
 # Graph Table
 
 action.GitMachete.DiscoverAction.text=Discover Branch Layout
-action.GitMachete.DiscoverAction.description=Discovers the Branch Tree in Git Machete Plugin
-action.GitMachete.DiscoverAction.cant-discover-layout-error-title="Can't discover repository branch tree"
-action.GitMachete.DiscoverAction.cant-get-current-repository-error-title="Can't get current git repository"
-action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.empty-table-text="No branch has been discovered"
+action.GitMachete.DiscoverAction.description=Discover Branch Tree in Git Machete Plugin
+action.GitMachete.DiscoverAction.cant-discover-layout-error-title=Can't discover repository layout
+action.GitMachete.DiscoverAction.cant-get-current-repository-error-title=Can't get current git repository
+action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.empty-table-text=No branch has been discovered
 action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-button-text=Save Discovered Layout
-action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.title=Discovered branch tree
+action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.title=Discovered branch layout
 action.GitMachete.DiscoverAction.repository-discover-error-title=Repository discover error
 action.GitMachete.DiscoverAction.write-file-task-title=Writing branch layout file
 action.GitMachete.DiscoverAction.write-file-error-title=Can't write discovered layout
 
+string.EnhancedGraphTable.empty-machete-file.upper-text=Provided machete file ({0}) is empty.
```

Comment:
Not sure about this `string` prefix... `table`?...

Reply:
I want to invent something universal that is appropriate not only for graph table but also for other future I18n messages that are not in actions

Reply:
Ok, let's leave `string`, see comment to lookbehinds though